src/components/common/button/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import { observer } from 'mobx-react';
+import React from 'react';
+
+import './button.scss';
+
+interface ButtonProps {
```

Comment:
some small changes to the props:

- rename onPlay to onClick
- exiting and loading should be optional
- add a className: string property - this gets added to the hardcoded 'button' classname in the comp